src/include/migraphx/op/rnn_variable_seq_lens.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_RNN_VARIABLE_SEQ_LENS_HPP
+#define MIGRAPHX_GUARD_OPERATORS_RNN_VARIABLE_SEQ_LENS_HPP
+
+#include <array>
+#include <migraphx/operation.hpp>
+#include <migraphx/op/common.hpp>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/stringutils.hpp>
+#include <migraphx/streamutils.hpp>
+#include <migraphx/literal.hpp>
+#include <migraphx/par_for.hpp>
+#include <migraphx/config.hpp>
+#include <cmath>
+#include <utility>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct rnn_shift_output
```

Comment:
Can we prefix all these operators with rnn_variable_` to make it clear its only used for variable sequences.

Reply:
how about rnn_var_sl_shift_sequence and rnn_var_sl_shift_output?

Reply:
renamed